Test/comp/Hello/Hello.php.expect

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
AFAICT this file is not actually used. The Hello.dfy lit test is only checking the output of the PHP program, not the source. You could add a separate assertion for the exact compiled source, but such a test is likely to be brittle and a pain to maintain.